ops/deploy.sh

```diff
@@ -131,6 +131,17 @@ kubectl --namespace=data-repo apply -f "${SCRATCH}/oidc-proxy-deployment.yaml"
 kubectl --namespace=data-repo apply -f "${SCRATCH}/cloudsql-proxy.yaml"
 kubectl --namespace=data-repo apply -f "${SCRATCH}/oidc-ingress.yaml"
 
+export SIDECAR_IMAGE_TAG=${SIDECAR_IMAGE_TAG:-'0.4.3'}
+export KUBE_NAMESPACE=${KUBE_NAMESPACE:-'default'}
```

Comment:
This could default to data-repo since default is never used with CIS compliancy.  

`export KUBE_NAMESPACE=${KUBE_NAMESPACE:-'data-repo'}`